bot/cogs/image_dots_cog.py

```diff
@@ -0,0 +1,199 @@
+import logging
+import urllib.request
+import random
+from PIL import Image, UnidentifiedImageError
+import math
+import requests
+
+import discord
+import discord.ext.commands as commands
+from bot.consts import Colors
+import bot.extensions as ext
+
+log = logging.getLogger(__name__)
+ASCIIYDOTS = 4
+ASCIIXDOTS = 2
+PC_WIDTH = 59
+MOBILE_WIDTH = 23
+# having this at the actual threshold of 2000 caused issues around
+# Just best to give some leniency
+CHAR_LIMIT = 1950
+class DotCog(commands.Cog):
+
+    def __init__(self, bot):
+        self.threshold = 150
+        self.inverted = 0
```

Comment:
we shouldnt use class level variables for command specific data, these should just be passed into the function